app/routes/helper_functions.py

```diff
@@ -0,0 +1,296 @@
+from flask import make_response, request, abort, jsonify
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from dotenv import load_dotenv
+from datetime import datetime
+from http import HTTPStatus
+from app import db
+import requests
+import os
+
+
+load_dotenv()
+
+
+NOWTIME = datetime.utcnow()
+SLACK_TOKEN = os.environ.get("SLACK_TOKEN")
+CHANNEL = os.environ.get("CHANNEL")
+
+
+def create_response(cls, instance, status_code=HTTPStatus.OK):
+    db.session.commit()
+    instance = instance.to_json()
+    cls_type = cls.__name__.lower()
+    return make_response({cls_type: instance}, status_code)
+
+
+def generate_error_message(cls, id):
```

Comment:
My one concern about this method is that it only describes one potential error. This particular method generates a message that always says the id is not found, which doesn't describe every error that could potentially pop up (Invalid input, missing information, etc). With that in mind, it might be better to simply create the error message in the moment as opposed to in a helper method.